README.md

```diff
@@ -1,13 +1,20 @@
 ember-cache-primitive-polyfill
 ==============================================================================
 
-[Short description of the addon.]
-
+Polyfills Ember's [cache primitive API](https://github.com/emberjs/rfcs/blob/master/text/0615-autotracking-memoization.md).
+
+```js
+import {
+  createCache,
+  getValue,
+  isConst
+} from '@glimmer/tracking/primitives/cache';
+```
 
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember.js v3.12 or above
+* Ember.js v3.13 or above
 * Ember CLI v2.13 or above
```

Comment:
Lets bump this to 3.8 or above (if you don't mind)

Reply:
we should probably bump that in the default blueprint

Reply:
Ya, I thought I did that on beta (for 3.19), but I’ll double check